[DOC-277] Migrate ESA playbook to Docusaurus page (#511)
Ed-Fi-Alliance-OSS/ed-fi-alliance-oss.github.io: docs/getting-started/esa-playbook/ed-fi-overview-and-esa-implementation-playbook.md

---
sidebar_position: 2
---

# Ed-Fi Overview and ESA Implementation

> This playbook covers how Educational Service Agencies (ESAs) can leverage Ed-Fi standards to provide data services for their member districts. The first section makes the case for Ed-Fi adoption; the second walks through the implementation journey.

Reviewer: ```suggestion
This playbook covers how Educational Service Agencies (ESAs) can leverage the Ed-Fi Data Standard to provide data services for their member districts. The first section makes the case for Ed-Fi Data Standard adoption; the second walks through the implementation journey.
```

| Education Service Agencies | Change-agent responsibilities, visibility into district data needs, and shared cost pressure. |
| Local Districts | Reporting burden (avg 6 staff at $0.5M per district), delayed absenteeism alerts, limited consolidated assessment views, and limited college/career readiness visibility. |

### Ed-Fi interoperability mission

Districts connect through an ESA to Ed-Fi, while states and vendors also connect directly. Ed-Fi provides three core supports: a data standard, open-source APIs, and community resources.

```mermaid
flowchart LR
  States["States"] --> EdFi["Ed-Fi Alliance<br/>Standards, APIs, Community"]
  Vendors["Vendors"] --> EdFi
  D1["District"] --> ESA["ESA"]
  D2["District"] --> ESA
  D3["District"] --> ESA
  ESA --> EdFi
  EdFi --> Standard["Data standard<br/>Common taxonomy for student-performance data"]
  EdFi --> APIs["Open-source APIs<br/>Real-time standardized data exchange"]
  EdFi --> Community["Community<br/>Knowledge base, vendor directory, support, events"]
```

### Michigan local-use-case examples

> Michigan examples show three categories of local impact enabled by Ed-Fi.

Reviewer: ```suggestion
Michigan examples show three categories of local impact enabled by the Ed-Fi Data Standard.
```

    Educator Prep
      Staff preparedness and availability
    Transcript Student Transfer
      Faster access to educational services
    Data Warehousing
      Downstream analytics and reporting
    Rostering
      OneRoster and similar applications
```

### Why ESAs are positioned to provide data services

ESAs occupy a unique position across existing market relationships, an expanded service opportunity, and the broader state/district/vendor ecosystem.

| Area | Description of impact |
| --- | --- |
| Existing Market | ESAs are involved in contracts and services, while districts struggle with data access and interoperability. |
| Your Opportunity | ESAs can offer a technology stack that helps districts now and supports layered ESA services later; the approach is stronger when ESAs collaborate and share resources. |
| Ecosystem | ESAs may not determine state direction, but they can enable district needs, include the state to drive vendor requirements, and reduce district burden. |

### Examples working today

South Carolina District Data Governance, Texas Education Exchange, and Michigan DataHub are examples of ESA/state data-service models operating today.

> ![Slide showing South Carolina District Data Governance, Texas Education Exchange, and Michigan DataHub as examples of Ed-Fi-related work already operating.](/img/esa-playbook/page-09.webp)

Reviewer: In this setting, the slide isn't adding any value. Let's link directly to the data hubs.

```suggestion
[Palmetto Data Collaborative](https://palmettodc.org/) (South Carolina), [Texas Education Exchange](https://txedexchange.net/), and [Michigan DataHub](https://www.midatahub.org/) are examples of ESA/state data-service models operating today.
```
